gma/src/android/ad_request_converter.cc

```diff
@@ -201,5 +201,26 @@ AdError MapAndroidFullScreenContentErrorCodeToCPPErrorCode(jint j_error_code) {
   }
 }
 
+AdError MapAndroidOpenAdInspectorErrorCodeToCPPErrorCode(int j_error_code) {
+  // Android AdInspector error codes sourced from:
+  // https://developers.google.com/android/reference/com/google/android/gms/ads/AdInspectorError#constants
+  switch (j_error_code) {
+    case 0:  // ERROR_CODE_INTERNAL_ERROR
```

Comment:
One thing that we could do is use the JNI Field stuff to actually read these values from the Android class. [Here's](https://github.com/firebase/firebase-cpp-sdk/blob/0c8c8b29bc2d62d66c6ac49ff2c3fb04f815a687/storage/src/android/storage_android.cc#L76) an example.